format/spec.md

```diff
@@ -193,10 +193,38 @@ Notes:
+| **`double`**       | **`number`**  | 1.1                                    |                                                                                                                                                                                                                                              |
+| **`decimal(P,S)`** | **`string`**  | "0x3162"                               | we use hexadecimal byte literals to encode bytes, with prefix `0x`, the byte array contain the two's-complement representation of the `unscaled` integer value in big-endian byte order, the actual value will be `unscaled * 10 ^ (-scale)` |
+| **`date`**         | **`number`**  | 19054                                  |                                                                                                                                                                                                                                              |
+| **`time`**         | **`number`**  | 36000000000                            |                                                                                                                                                                                                                                              |
+| **`timestamp`**    | **`number`**  | 1646277378000000                       |                                                                                                                                                                                                                                              |
+| **`timestamptz`**  | **`number`**  | 1646277378000000                       |                                                                                                                                                                                                                                              |
+| **`string`**       | **`string`**  | "foo"                                  |                                                                                                                                                                                                                                              |
+| **`uuid`**         | **`string`**  | "eb26bdb1-a1d8-4aa6-990e-da940875492c" |                                                                                                                                                                                                                                              |
+| **`fixed(L)`**     | **`string`**  | "0x3162"                               | we use hexadecimal byte literals to encode bytes, with prefix `0x`                                                                                                                                                                           |
```

Comment:
All text, including notes or column headings, should use sentence case. That is, the first letter should be capitalized. There is no need for complete sentences; be as direct and concise as possible while being clear.